### ISSUE TITLE
Infinite recursive call .lorem on Optional property type

### ISSUE THREAD

**Issue by sergeymild** [EnesKaraosman/LoremSwiftify#10]
```
@LoremSwiftify
public struct Job: Codable {
    @Lorem(.url(.website))
    public let url: String?
    public let company: String?
    public let job: String
}
```
in this case there will be infinite call in this place
```
extension LoremIpsumize {
    public static func lorem(_ kind: LoremKind? = nil) -> Self {
        lorem(kind)
    }
}
```

if add to Optional extension
```
extension Optional: LoremIpsumize {
    public static func lorem() -> Optional<Wrapped> {
        .none
    }
    
    public static func lorem(_ kind: LoremKind?) -> Optional<Wrapped> {
        return .none
    }
}
```
error is gone but value will be nil in result

**Comment by EnesKaraosman** [comment]
Hi @sergeymild. Thanks for reporting. I'll check when available..

**Comment by EnesKaraosman** [comment]
Should be fixed with the new release 0.2.1 @sergeymild
